### DISCUSSION TITLE
blog/nextjs/4-nextjs-13ver

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#82] [General]
# blog/nextjs/4-nextjs-13ver

Next.js의 13 버전에서 달라진 내용과 이를 잘 활용할 수 있는 방법을 알아보겠습니다.

https://enjoydev.life/blog/nextjs/4-nextjs-13ver

**Comment by meowTarae** [comment]
13 버전의 변경점을 잘 정리해 주셨네요!
쉽게 잘 설명해주셔서 이해하기에도 너무 좋았어요 👍

**Reply by pySoo** [reply]
아이쿠 타래님 여기서 뵙게 되다니 ㅎㅎ
도움이 됐다니 다행입니다..! 읽어주셔서 감사해요 😸
